content/sensu-go/5.13/reference/assets.md

```diff
@@ -96,6 +96,186 @@ sensu-example-handler_1.0.0_linux_amd64
 └── include
 {{< /highlight >}}
 
+### Asset hello world example
+
+In this example, we'll run a script that outputs `Hello World`:
+
+{{< highlight bash >}}
+hello-world.sh
+
+#!/bin/sh
+
+STRING="Hello World"
+
+echo $STRING
+
+if [ $? -eq 0 ]; then
+    exit 0
+else
+    exit 2
+fi
+{{< /highlight >}}
+
+The first step is to ensure that our directory structure is in place. As noted in [Example structure](#example-structure), our script could live in three potential directories in the project: `/bin`, `/lib`, or `/include`. For this case, we will put our script in the `/bin` directory. We'll create the directories `sensu-go-hello-world` and `/bin`:
+
+{{< highlight bash >}}
+$ mkdir sensu-go-hello-world
+
+$ cd sensu-go-hello-world
+
+$ mkdir bin
+
+$ tree
+.
+└── bin
+    └── hello-world.sh
+{{< /highlight >}}
+
+Next, we'll create our script in the `/bin` directory and ensure that it's marked as executable:
+
+{{< highlight bash >}}
+$ chmod +x bin/hello-world.sh 
+mode of 'hello-world.sh' changed from 0644 (rw-r--r--) to 0755 (rwxr-xr-x)
+{{< /highlight >}}
+
+Now that the script is in the directory, let's move on to the next step: packaging the asset.
+
+#### Packaging the asset
+
+Assets are archives, so the first step in packaging the asset is to create a tar.gz archive of our project. This assumes we're in the directory we want to tar up:
+
+{{< highlight bash >}}
+$ cd ..
+$ tar -C sensu-go-hello-world -cvzf sensu-go-hello-world-0.0.1.tar.gz .
+...
+{{< /highlight >}}
+
+Excellent. Now that we've created an archive, we'll need to generate a SHA512 sum for it (this is required--otherwise, the asset won't work):
+
+{{< highlight bash >}}
+sha512sum sensu-go-hello-world-0.0.1.tar.gz | tee sha512sum.txt
+dbfd4a714c0c51c57f77daeb62f4a21141665ae71440951399be2d899bf44b3634dad2e6f2516fff1ef4b154c198b9c7cdfe1e8867788c820db7bb5bcad83827 sensu-go-hello-world-0.0.1.tar.gz
+{{< /highlight >}}
+
+Now that we have our sha512sum, we'll need to host the release (archive and sha512sum) somewhere. You can do this with S3, a GitHub release, or even just serving the files out of a directory using Nginx/Apache.
+
+In this case, we'll use GitHub to serve our release. First, let's create a release: click **Create a new release**: 
+
+![Create a new github release][gh-release-01]
+
+We will see the following screen:
+
+![Release details screen][gh-release-02]
+
+On this screen, we'll enter a tag version, release title, and release description details and drag and drop our asset and checksum so they will be uploaded as part of the release. When we've done that, the screen will look something like this:
+
+![Release details filled in][gh-release-03]
+
+Click **Publish release**. The latest release page will load:
+
```

Comment:
Looks like the screenshot cut off the **Publish release** button. It might be worth re-taking the screenshot so that it shows that button.

Reply:
👍